src/reducers/CurrentOpenFileState.js

```diff
@@ -29,17 +32,39 @@ const initialFileName = {
   viewName: storedInitialFileName.hasOwnProperty('viewName') ? storedInitialFileName.viewName : '',
 }
 
+const emptyFileId = {
+  sourceId: NO_OPEN_FILE_ID.sourceId,
+  viewId: NO_OPEN_FILE_ID.viewId,
+  viewType: NO_OPEN_FILE_ID.viewType,
+}
+
+const emptyFileName = {
+  sourceName: '',
+  viewName: ''
+}
+
 const SET_CURRENT_OPEN_FILE_ID_ACTION_TYPE = 'currentOpenFileId/set';
 export const setCurrentOpenFileIdAction = fileId => ({ type: SET_CURRENT_OPEN_FILE_ID_ACTION_TYPE, fileId });
-export const currentOpenFileIdReducer = (state = initialFileId, action) =>
-  action.type !== SET_CURRENT_OPEN_FILE_ID_ACTION_TYPE || !validateHasFileIdObj(action)
+export const currentOpenFileIdReducer = (state = initialFileId, action) => {
+  if (action.type === SIGNOUT){
+    console.log("setting to empty file id");
+    return emptyFileId
+  }else{
```

Comment:
huh... how did this pass local formatting?!

Reply:
ah I see the formatter broke in the last PR. It's fine - we can reformat all the code once this is merged